save-backend/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ import  org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     kotlin("jvm")
+    id("org.jetbrains.kotlin.plugin.jpa") version Versions.kotlin
```

Comment:
I believe, `all-open` plugin is needed for entities as well.

Reply:
Added.